war/src/main/webapp/css/style.css

```diff
@@ -1975,3 +1975,23 @@ body.main-panel-only #main-panel {
         padding: 2em 10%;
     }
 }
+
+IMG.icon-sm {
```

Comment:
You youngsters don't seem to know, but historically [canonical tag names have been upper case](https://www.w3.org/TR/html4/index/elements.html). In HTML5 the canonical tag names seem to have switched to [lower case](https://www.w3.org/TR/html5/index.html) though.